src/components/atoms/Timeline/index.tsx

```diff
@@ -35,15 +52,76 @@ const Timeline: React.FC<Props> = memo(
       strongScaleHours,
       currentPosition,
       events,
+      player: {
+        playSpeed,
+        onChangePlaySpeed,
+        formattedCurrentTime,
+        isPlayed,
+        isPlayBacked,
+        toggleIsPlayed,
+        toggleIsPlayBacked,
+      },
     } = useTimeline({
       currentTime,
       range,
       onClick,
       onDrag,
+      onPlay,
     });
+    const t = useT();
+
+    if (!isOpened) {
+      return (
+        <OpenButton onClick={onOpen}>
+          <Icon alt={t("Open timeline")} icon="timeline" size={24} />
+        </OpenButton>
+      );
+    }
```

Comment:
In Re:Earth, usually when we have conditional rendering we will just do a ternary operator in the main return.
In this case it'd be like
```
return isOpened ? (
<Container>
...
...
</Container>) : (
   <OpenButton onClick={onOpen}>
          <Icon alt={t("Open timeline")} icon="timeline" size={24} />
   </OpenButton>
)
```

Reply:
This was fixed by [refactor: use conditional operator](https://github.com/reearth/reearth-web/pull/277/commits/f9e3a78b36e4bf9a7c785367a43b05592e23d794)